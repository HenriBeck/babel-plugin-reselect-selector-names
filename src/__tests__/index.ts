import * as babel from '@babel/core';
import plugin from '../';

it('works', () => {
  const res = babel.transform(
    `
  const getConfig = createSelector();
  `,
    { plugins: [plugin] },
  );

  expect(res && res.code).toMatchSnapshot();
});
