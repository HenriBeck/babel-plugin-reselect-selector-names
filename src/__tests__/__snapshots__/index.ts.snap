// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`works 1`] = `
"const getConfig = createSelector();
getConfig.selectorName = \\"getConfig\\";"
`;
